src/utils/getSearchData.js

```diff
@@ -0,0 +1,7 @@
+const getSearchData = (filteredData, searchInput) => {
+  return filteredData.filter(
+    (item) =>
+      item.modelName.toLowerCase().indexOf(searchInput.toLowerCase()) !== -1
```

Comment:
Maybe use varibale for `item.modelName.toLowerCase()` and `searchInput.toLowerCase()`